cmd/cleanup.go

```diff
@@ -60,6 +63,7 @@ func newCleanupCmd(out io.Writer) *cobra.Command {
 	settings.AddFlags(flags)
 
 	flags.BoolVar(&configCleanup, "config-cleanup", false, "if set, configuration cleanup performed")
+	flags.StringVarP(&releaseName, "name", "n", "", "the release name. When it is specified, the named release and its versions will be removed only. Should not be used with other cleanup operations")
```

Comment:
I think the short form -n will not work because it is already used for the short form of --namespace as a global flag to helm.

Reply:
we just could use then `--name` only

Reply:
Updated removing the short form `-n`.